src/ui/component/transactionList/view.jsx

```diff
@@ -103,7 +102,7 @@ class TransactionList extends React.PureComponent<Props> {
             </div>
           )}
         </header>
-        {!transactionList.length && <p>{emptyMessage || __('No transactions to list.')}</p>}
+        {!transactionList.length && <p className="card__subtitle">{emptyMessage || __('No transactions to list.')}</p>}
```

Comment:
If there's not a style for empty lists/cards/elements, can we add one? Probably more for @NetOperatorWibby than @seanyesmunt